### DISCUSSION TITLE
blog/a-comprehensive-guide-to-data-fetching-in-nuxt-3

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#19] [Announcements]
# blog/a-comprehensive-guide-to-data-fetching-in-nuxt-3

With Nuxt 3's rendering modes, you can execute API calls and render pages both on the client and server, which has some challenges. For example, we want to avoid duplicate network calls, efficient caching, and ensure that the calls work across environments. To address these challenges, Nuxt provides a built-in data fetching library ($fetch) and two composable (useFetch and useAsyncData).

https://mokkapps.de/blog/a-comprehensive-guide-to-data-fetching-in-nuxt-3

**Comment by jswhisperer** [comment]
Was going to ask about Abort fetch support but found the docs, it seems supported https://vueuse.org/core/useFetch/#aborting-a-request 
Cheers great indepth article!
